### PR TITLE
Add select

### PR DESCRIPTION
**Pull request by JacobBlomgren**

### CODE REVIEW

File: src/Molecules/Select/Select.tsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  className,
+  value: valueFromProps,
+  onChange: onChangeFromProps,
+  onBlur: onBlurFromProps,
+  onFocus: onFocusFromProps,
+}) => {
+  const [focus, setFocus] = useState(false);
+  const [value, setValue] = useState();
+
+  const onChange = e => {
```
Comment (JacobBlomgren) [inline src/Molecules/Select/Select.tsx]: just need typescript help here @shmidt-i

**Comment by mjmonline** [comment]
Great!  🙌

**Comment by mjmonline** [comment]
We might need to have a prop to make the select higher/bigger

File: src/Molecules/Select/Select.tsx (hunk 1 above)
Comment (jauniuse) [reply src/Molecules/Select/Select.tsx]: I suppose e is of type `React.ChangeEvent<HTMLSelectElement>`

**Pull request merged**